allennlp/modules/token_embedders/embedding.py

```diff
@@ -140,6 +149,25 @@ def forward(self, inputs):  # pylint: disable=arguments-differ
             embedded = projection(embedded)
         return embedded
 
+    @overrides
+    def extend_vocab(self, extended_vocab: Vocabulary, vocab_namespace: Optional[str] = None):
```

Comment:
The `Optional` annotation on `vocab_namespace` is redundant, because you immediately follow it with ` = None`, which means exactly the same thing.  Just do `vocab_namespace: str = None`.

Reply:
Ok, sure. I've made this mistake a couple of places. Will fix them.